### DISCUSSION TITLE
Why `mingw-w64-efl` only available for `MINGW64`?

### DISCUSSION THREAD

**Discussion by ghost** [msys2/MINGW-packages#19590] [General]
https://packages.msys2.org/base/mingw-w64-efl

**Comment by MehdiChinoune** [comment]
Bacause, It doesn't build for other environments. 
Someone has to fix it.

**Reply by ghost** [reply]
It doesn't build on `UCRT64`? Isn't `UCRT64` is a drop-in replacement to `MINGW64`?

**Reply by ZachBacon** [reply]
> It doesn't build on `UCRT64`? Isn't `UCRT64` is a drop-in replacement to `MINGW64`?

Nope, just uses a different runtime than mingw-w64

**Reply by MehdiChinoune** [reply]
There are few packages that don't build on UCRT64, About 10 packages.

**Reply by MehdiChinoune** [reply]
The original author chose to not support ucrt64 https://github.com/msys2/MINGW-packages/pull/17484/files

**Comment by ghost** [comment]
> > It doesn't build on `UCRT64`? Isn't `UCRT64` is a drop-in replacement to `MINGW64`?
> 
> Nope, just uses a different runtime than mingw-w64

As I know, the MinGW-w64 approach makes it a drop-in replacement to the legacy `MSVCRT`. You can't find any import libraries for UCRT as they are reusing the old facility for `MSVCRT` to support the newer `UCRT`. The import library is still the same!

**Comment by MehdiChinoune** [comment]
It's now available for UCRT64 and CLANG64 https://github.com/msys2/MINGW-packages/commit/216dbaf8fa284663cd3902a0306147314371b54c
